src/__tests__/tsconfig-loader.test.ts

```diff
@@ -167,262 +164,135 @@ describe("walkForTsConfig", () => {
   });
 });
 
-describe("loadConfig", () => {
+describe("loadSyncDefault", () => {
   it("should load a config", () => {
-    const config = { compilerOptions: { baseUrl: "hej" } };
-    const res = loadTsconfig(
-      "/root/dir1/tsconfig.json",
-      (path) => path === "/root/dir1/tsconfig.json",
-      (_) => JSON.stringify(config)
-    );
-    expect(res).toStrictEqual(config);
+    const cwd = resolve(__dirname, "../../example/basic")
+
+    const result = tsConfigLoader({ cwd, getEnv: () => undefined });
+
+    expect(result).toEqual({
+      baseUrl: "./hej",
+      paths: {},
+      tsConfigPath: resolve(cwd, "tsconfig.json"),
+    });
   });
 
   it("should load a config with comments", () => {
-    const config = { compilerOptions: { baseUrl: "hej" } };
-    const res = loadTsconfig(
-      "/root/dir1/tsconfig.json",
-      (path) => path === "/root/dir1/tsconfig.json",
-      (_) => `{
```

Comment:
since the API from `get-tsconfig` does not support a resolve callback like this, I opted instead to create entries in the `example/` folder for each test case